WalletWasabi/Crypto/StrobeProtocol/Strobe.cs

```diff
@@ -133,6 +133,7 @@ private void BeginOperation(StrobeFlags flags, bool more)
 					throw new InvalidOperationException($"Attempt to continue operation '{_currentFlags}' with new flags '{flags}' is not allowed.");
 				}
 				_currentFlags = flags;
+				return;
```

Comment:
This is intentional right?

Reply:
Yes, that is there to allow adding data in more than just one call. In fact, if you remove that return the tests will fail.